src/preFilter.ts

```diff
@@ -18,14 +19,18 @@ export function filterByContent(searchee: Searchee): boolean {
 		});
 	}
 
-	const isSingleEpisodeTorrent =
-		searchee.files.length === 1 && EP_REGEX.test(searchee.files[0].name);
-
-	if (!includeEpisodes && isSingleEpisodeTorrent) {
+	const isSingleEpisodeTorrent = searchee.files.length === 1 && EP_REGEX.test(searchee.name); 
+	const isSeasonPack = typeof searchee.path !== 'undefined' ? SEASON_REGEX.test(path.dirname(searchee.path).split(path.sep).reverse()[0]): false;
```

Comment:
this dirname -> split -> reverse -> [0] thing could be changed to dirname -> basename

```ts
basename(dirname('/a/b/c/d')) === 'c'
```